dotcom-rendering/src/web/components/PullQuoteBlockComponent.tsx

```diff
@@ -204,6 +204,54 @@ export const PullQuoteBlockComponent: React.FC<{
 					</footer>
 				</aside>
 			);
+		case ArticleDesign.Analysis:
+			return (
+				<aside
+					css={[
+						decidePosition(role, design),
+						css`
+							${headline.xxsmall({ fontWeight: 'bold' })};
+							line-height: 25px;
+							position: relative;
+							background-color: ${neutral[100]};
+							padding-left: 10px;
+							padding-right: 10px;
+							padding-top: 6px;
+							padding-bottom: 12px;
+							margin-bottom: 1.75rem;
+							color: ${palette.text.pullQuote};
+
+							:after {
+								content: '';
+								width: 25px;
+								height: 25px;
+								bottom: -25px;
+								position: absolute;
+								background-color: ${neutral[100]};
+							}
+						`,
+					]}
+				>
+					<QuoteIcon colour={palette.fill.quoteIcon} />
```

Comment:
I think this colour here should match the background of the main square? It looks like there's different based on the screenshot

I think this file needs a little updating for this to work but ideally we'd have all this colour variation managed by decidePalette, and then we'd add any variation for `Design.Analysis` there

Reply:
Thanks for the tips! Will action.